Marlin/src/lcd/language/language_it.h

```diff
@@ -251,6 +251,8 @@ namespace Language_it {
   PROGMEM Language_Str MSG_CONTROLLER_FAN_AUTO_ON          = _UxGT("Modo autom.");
   PROGMEM Language_Str MSG_CONTROLLER_FAN_SPEED            = _UxGT("Vel. attivo");
   PROGMEM Language_Str MSG_CONTROLLER_FAN_DURATION         = _UxGT("Tempo inattivo");
+  PROGMEM Language_Str MSG_FAN_ON                          = _UxGT("Fan on");
```

Comment:
If it's meant to be an action:
Fan On=Accendi ventola
Fan off=Spegni ventola
If it's meant to be a status:
Fan On=Ventola accesa
Fan off=Ventola spenta

Reply:
GitHub now allows you to add "change requests" directly when viewing a file. If the author accepts the change request then it is applied as a new commit from you.